website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -4,7 +4,7 @@ description: "Use exports to materialize tables to the data platform on a schedu
 sidebar_label: "Materialize with exports"
 ---
 
-# Materialize queries with exports <Lifecycle status='new'/>
+### Materialize queries with exports <Lifecycle status='new' />
```

Comment:
sugg: i've been taught that it's bad practice to skip heading levels (ie use them in order)

```suggestion
# Materialize queries with exports <Lifecycle status='new' />
```

Reply:
oh dear, this was a mistake! i was testing the issue you had with the lifecycle in the separate pr and also had a failed deploy. but i forgot to remove the h3 🙈 

thank you so much for the catch!